assets/translations/en-us.yaml

```diff
@@ -1073,6 +1073,27 @@ cluster:
       =1 {is 1 Harvester cluster that is not shown}
       other {are # Harvester clusters that are not shown}
       }  
+    registration:
+      step1: "1. Go to the settings page of Harvester UI."
```

Comment:
Now I understand the issue, I think this should be updated to 

`1. Go to the settings page of the target Harvester's UI`

Reply:
Done